### ISSUE TITLE
Compatibility Windows server 2016

### ISSUE THREAD

**Issue by ghost** [hathoward/windows_firewall#9]
Dear developer,

I want to use your forge module, since it's able to purge unmanaged firewall rules and thus is able to maintain a kind of 'desired state configuration'.

Unfortunately, it's incompatible with Windows Servers 2016.

Can this be added?

Thanks in advance.

with kind regards,
Tom

**Comment by Romiko** [comment]
Yes, we would love support for Windows 2016. It is possible we can Fork this Repo and extend the module? 

The other aspect we would love to have is the ability for the module to recognize "SYSTEM" firewall rules or other rules tied to an application name e.g. svchost.exe, chrome.exe etc.  and to either  ignore/disable them, but never delete/purge them.

**Comment by thatrevguy** [comment]
It's been some time since I've worked on this. Please feel free to fork. Would like to revisit this to automate PR tests and builds. As for 2016 support, there's a case statement that is used to check for "compatibility" here https://github.com/thatrevguy/windows_firewall/blob/master/manifests/init.pp#L13

Do not have a 2016 environment to test against to validate that this is all that requires an update. As long as wmi has not been altered too much it should be OK.
